src/humanPose/index.js

```diff
@@ -240,7 +240,7 @@ import {Pose} from "./Pose.js";
                     return;
                 }
                 if (length !== JOINTS_PER_POSE) {
-                    if (length == JOINTS_V1_COUNT) {
+                    if (length == JOINTS_V1_COUNT || length == JOINTS_V2_COUNT) {
                         utils.convertFromJointsV1(jointPositions, jointConfidences);
```

Comment:
Would be nice to have a specific convertFromJointsV2 function (even if it just calls convertFromJointsV1 and does nothing else) for clarity of use